src/hevm/hevm.cabal

```diff
@@ -108,7 +108,7 @@ library
   autogen-modules:
     Paths_hevm
   ghc-options:
-    -Wall -Wno-deprecations -Wno-unticked-promoted-constructors
+    -Wall -Wno-deprecations -Wno-unticked-promoted-constructors -j6
```

Comment:
Why 6?

Reply:
I think we can just use `-j` and then it will figure out how many cores to use based on the machine that it's executing on?

Reply:
Ah, new feature! Finally they don't refuse to guess the number of processors :D This has been bugging me for about 10 years, but finally, they did fix it! It used to be that `-j` launched as many processes as there were compile items... so it'd crash immediately on large projects. Finally fixed! Yep, OK, I am moving to `-j`, apparently it now works:

`
Using the [-j[⟨n⟩]](https://downloads.haskell.org/ghc/latest/docs/users_guide/using.html#ghc-flag--j[%E2%9F%A8n%E2%9F%A9]) flag, you can compile modules in parallel. Specify -j ⟨n⟩ to compile ⟨n⟩ jobs in parallel. If ⟨n⟩ is omitted, then it defaults to the number of processors.
`
OK, I'm fixing this :)

Reply:
Fixed to `-j`. Thanks for the feedback!